shared/src/commonMain/kotlin/org/hyperskill/app/likes/domain/model/LikeValue.kt

```diff
@@ -1,6 +1,5 @@
 package org.hyperskill.app.likes.domain.model
 
 enum class LikeValue(val value: Int) {
-    VALUE_0(0),
     VALUE_1(1)
```

Comment:
`VALUE_1 ` is used when we call `abuseComment`. Let's add a human-readable comment about what it means, for example "Means dislike".